modules/node_modules/@ncigdc/components/GitHut.js

```diff
@@ -60,11 +77,20 @@ class GitHut extends React.Component {
         $state.go = this.handleAngularRouterRequestGo;
         LocationService.filters = () => this.props.params.filters;
       }])
-      .directive('githut', () => ({
+      .directive('githut', ['$timeout', ($timeout) => ({
         controller: 'ProjectsController as prsc',
         link(scope, element, attrs, controller) {
           controller.ProjectsState.tabs.graph.active = true;
           self.angularController = controller;
+
+          // NOTE: bandaid solution to hide duplicate primary sites created due to projects now having multiple primary sites
+          $timeout(() => {
+            const duplicatePrimarySiteTexts = _.flatten(Object.values(_.groupBy(element.find('.githut #pc svg #labels .label:last-child text'), 'innerHTML')).filter(els => els.length > 1).map(_.tail));
+            duplicatePrimarySiteTexts.forEach(text => {
+              text.style.fillOpacity = 0;
+              text.style.pointerEvents = 'none';
+            });
+          }, 3000);
```

Comment:
Not without trading off on how certain we are that the graph is done rendering, or ideally figuring out an alternative to $timeout (I did give up fairly quickly (under the rationale that the consequence is that a few words would look bolder for a few seconds longer than ideal))